server/navigation/_manual_routes.py

```diff
@@ -0,0 +1,69 @@
+# Gives the initial piece of the dictionary where we have to manually enter routes and endpoints
+def get_manual_routes():
```

Comment:
rather than hard-code this, would it make sense to make this a config file? Probably in .json format?